<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderDlg</class>
 <widget class="QDialog" name="FolderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select folder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qHostMonitor.qrc">
    <normaloff>:/img/hostMonitor.png</normaloff>:/img/hostMonitor.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnCopy">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>34</height>
          </size>
         </property>
         <property name="icon">
          <iconset resource="../qHostMonitor.qrc">
           <normaloff>:/img/action/copy.png</normaloff>:/img/action/copy.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEdit">
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>34</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>34</height>
          </size>
         </property>
         <property name="icon">
          <iconset resource="../qHostMonitor.qrc">
           <normaloff>:/img/action/properties.png</normaloff>:/img/action/properties.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDisable">
         <property name="minimumSize">
          <size>
           <width>36</width>
           <height>34</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>34</height>
          </size>
         </property>
         <property name="icon">
          <iconset resource="../qHostMonitor.qrc">
           <normaloff>:/img/action/disableTest.png</normaloff>:/img/action/disableTest.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qHostMonitor.qrc"/>
  <include location="../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FolderDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FolderDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
